### DISCUSSION TITLE
Bluetooth Low Energy Interface

### DISCUSSION THREAD

**Discussion by R-J** [organicmaps/organicmaps#1562] [Ideas]
It would be great if it would be possible to use external devices with the app over BLE like Komoot realized it with their [BLE Connect](https://www.komoot.com/b2b/connect#bleconnect) approach. For bike rides, projects which implement an e-ink + esp32 turn by turn navigation are very attractive. 

If exactly that standard would be implemented, existing devices could be reused.

**Comment by biodranik** [comment]
Is there an open BLE standard for such apps/interfaces/devices? Doing something proprietary is not fun.

**Reply by R-J** [reply]
I don't think it's an open standard, but you can take a look at [their documentation](https://docs.google.com/document/d/1iYgV4lDKG8LdCuYwzTXpt3FLO_kaoJGGWzai8yehv6g/edit#heading=h.n23mblj5jslk). But that's a very good point! If there is a standard to rely on, that would certainly be the best approach!

I found a [question on StackOverflow](https://stackoverflow.com/questions/69809003/is-there-a-standard-way-of-sending-gps-navigation-over-ble) which has no satisfying answer. The official [bluetooth specs](https://www.bluetooth.com/specifications/specs/) also do not list a service like that. I did some research now but it seems as if there is no standard for sending turn by turn navigation to bluetooth devices (not even some proposal, I'd say)
